babel/core.py

```diff
@@ -60,6 +60,9 @@ def get_global(key):
             fileobj.close()
     return _global_data.get(key, {})
 
+# FIXME: Very ugly hack in order to break cyclic import dependency
+localedata.get_global = get_global
```

Comment:
Hey, tbh a cyclic import smells. I don't have time to look into details of this one right now but usually it shows that a refactoring is needed.